src/validation.h

```diff
@@ -492,6 +492,12 @@ class CChainState
     //! Manages the UTXO set, which is a reflection of the contents of `m_chain`.
     std::unique_ptr<CoinsViews> m_coins_views;
 
+    //! This toggle exists for use when doing background validation for UTXO
+    //! snapshots. It is set once the background validation chain reaches the
+    //! same height as the base of the snapshot, and signals that we should no
+    //! longer connect blocks to this chainstate.
```

Comment:
In commit "validation: add CChainState::m_stop_use and ChainMan::isUsable" (c703836160f3eca8073c22a1531bb98ad9666999)

Would `s/this chainstate/the background chainstate/`. Which chainstate is "this chainstate" was not immediately obvious.

Reply:
Fixed